Exercise1 sql.txt

```diff
@@ -0,0 +1,61 @@
+--create 'student_db' database
```

Comment:
Reserved keyword must be capital follow the syntax properly.

Reply:
ok, will follow the capital keyword syntax now onwards